opta/cli.py

```diff
@@ -61,6 +63,7 @@ def cli() -> None:
         # after the command.
         # However, we should still clean them up before the next command, or
         # else it may interfere with it.
+        meister.REPORT_ENABLED = VERSION in [DEV_VERSION, ""]
```

Comment:
Let's make this based on an env flag instead. I don't want outside contributors to run into this.